eventkit_cloud/settings/prod.py

```diff
@@ -202,3 +202,5 @@
 }
 
 DISABLE_SSL_VERIFICATION = os.environ.get('DISABLE_SSL_VERIFICATION', False)
+
+MAX_EXPORTRUN_EXPIRATION_DAYS = 45
```

Comment:
This setting needs to be updateable via the environment `os.environ.get`, and let's default it to 30 days.